### PR TITLE
[0.9.x] RHBPMS-1889: Search by Find form does not work

### PR DESCRIPTION
**Pull request by manstis**
Fix failing tests on kie-wb-common and include test in uberfire.

### CODE REVIEW

**Comment by manstis** [comment]
This is related to https://issues.jboss.org/browse/RHBPMS-1889 and https://github.com/uberfire/uberfire-extensions/pull/312

**Comment by manstis** [comment]
@psiroky This fixes the failing tests on ```kie-wb-common``` (6.5.x)

File: uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/IOServiceIndexedDeleteFileTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                                  return "delete";
+                              }
+
+                              @Override
+                              public Object value() {
+                                  return "me";
+                              }
+                          });
+
+        Thread.sleep(5000); //wait for events to be consumed from jgit -> (notify changes -> watcher -> index) -> lucene index
```
Comment (ederign) [inline uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/IOServiceIndexedDeleteFileTest.java:61]: In future, we should refactoring lucene index api to trigger a signal when the index is ready!

**Review by ederign — APPROVED** [review]

**Comment by ederign** [comment]
Thanks @manstis

**Pull request merged**